wiki/Help_centre/Upgrading_to_lazer/vi.md

```diff
@@ -0,0 +1,288 @@
+
+#### Còn gì nữa?
+
+Chúng tôi có một lượng lớn các tính năng và cải tiến do người dùng yêu cầu mà chúng tôi sẽ tiếp tục phát triển với tốc độ ánh sáng. Đối với những người đã tham gia với chúng tôi gần đây và chưa trải nghiệm động lực của osu! phát triển, chuẩn bị để được ở trong một bất ngờ.
+
+#### Làm cách nào để truy cập thư mục bài hát?
+
+Không có thư mục bài hát trong lazer! Điều này cho phép chúng tôi làm những điều thú vị như không yêu cầu nhấn `F5` khi chọn bài hát để làm mới beatmap (vì beatmap luôn ở trạng thái tốt) và giảm 20–40% dung lượng ổ đĩa được sử dụng bởi beatmap. Bạn có thể đọc thêm về [cách lazer lưu trữ tệp](/wiki/Client/Release_stream/Lazer/File_storage).
+
+Nếu bạn cần thay đổi beatmap, vui lòng sử dụng trình chỉnh sửa. Trong tương lai, chúng tôi sẽ giới thiệu một chế độ trong trình chỉnh sửa giúp thư mục của beatmap có thể truy cập tạm thời để chỉnh sửa bên ngoài. Điều này sẽ cho phép bạn sử dụng các công cụ bên ngoài trên beatmap trong quá trình tạo.
+
+#### Bây giờ "osu!direct" đã có sẵn cho tất cả người chơi, liệu những người ủng hộ có bất kỳ lợi ích mới nào không?
+
+Một số bộ lọc trong danh sách beatmap vẫn chỉ dành cho người hỗ trợ.
+
+Ngoài ra còn có một số lợi ích bổ sung:
+
+- Những người ủng hộ có thể tạo danh sách phát kéo dài hơn
+
+Chúng tôi dự định sẽ xem xét các lợi ích mới trong tương lai, nhưng trọng tâm của chúng tôi hiện tại là tính năng tương đương với tính năng ổn định, vì vậy vui lòng sử dụng giao dịch mua thẻ người ủng hộ của bạn như một cách để... hỗ trợ sự phát triển của trò chơi!
+
+#### Nếu tôi gian lận trên lazer thì tôi có bị  không?
+
+Đương nhiên rồi.
+
+#### Nếu tôi phát hiện ai đó gian lận trên lazer tôi nên báo cáo họ như thế nào?
+
+Y như cách mà bạn vẫn hay làm thôi.
+
+#### Giao dịch vi mô ở đâu?
+
+Có vẻ như bạn đang nghĩ về một trò chơi khác.
+
+### Ghi chú
+
+[^wine]: Sử dụng wine.
+[^compatibility-mode]: DirectX thông qua chế độ tương thích.
+[^coming-soon]: Sắp có.
+[^dll]: Thủ công thông qua các tệp `.dll`.
+[^share-files]: Beatmap và skin sẽ chia sẻ tệp và tiết kiệm dung lượng ổ đĩa.
+[^gameplay-only]: Chỉ riêng gameplay.
+[^online]: Thông qua truy xuất trực tuyến.
+[^normalisation]: Điều này chỉnh các màu combo của beatmap đến cùng một mức độ sáng.
+[^hold-for-hud]: Giữ `Ctrl` để xem HUD trong giây lát khi nó bị ẩn.
+[^offset-calibration-stable]: Điều chỉnh thủ công thông qua các phím.
+[^offset-calibration-lazer]: Khi thử lại beatmap, bạn có thể hiệu chỉnh độ lệch dựa trên lần chơi cuối cùng của mình.
+[^can-disable]: Có thể bị vô hiệu hóa.
+[^note-lock]: Vẫn tồn tại, nhưng không nên can thiệp.
+[^score-reset-balance]: Điểm số sẽ được thiết lập lại để đảm bảo tính cân bằng.
+[^score-reset-isolated]: Điểm số sẽ được đặt lại, hiện đang bị cô lập khỏi điểm số ổn định.
+[^online-content]: Truy cập gốc vào hầu hết nội dung trực tuyến.
```

Comment:
```suggestion
[^online-content]: Truy cập gốc vào hầu hết nội dung trực tuyến.
```
Needs other suggestions about "native access"

Reply:
I deleted the "gốc"

Reply:
```suggestion
[^online-content]: Truy cập trực tiếp vào hầu hết nội dung trực tuyến.
```
pretty sure its the same as direct access